lib/salus/package_utils/gradle_dependency_parser.rb

```diff
@@ -1,16 +1,11 @@
 module Gradle
-  GRADLE7 = "/opt/gradle/gradle-7.3.3/bin/gradle".freeze
-  GRADLE6 = "/opt/gradle/gradle-6.9.2/bin/gradle".freeze
-
   def gradle_dependencies
     dependency_metadata_regex = /-\s(?<group_id>.+):(?<artifact_id>.+):(?<version>.+)/
 
     # 'gradle dependencies' command needs to be run in the folder where buid.gradle is present.
-    shell_result = run_shell("#{GRADLE7} dependencies")
-
-    shell_result = run_shell("#{GRADLE6} dependencies") if !shell_result.success?
+    shell_result = run_shell("/home/repo/gradlew dependencies")
     if !shell_result.success?
```

Comment:
Does `gradlew` follow the standard exit codes, like 0 on success and something else on failure.

Reply:
it follows the standard exit codes. Right now they only have 0 on success and 1 on failure. [According to this issue](https://github.com/gradle/gradle/issues/12374) They may be adding more exit codes soon to make them more descriptive